dyscover/dyscover.py

```diff
@@ -0,0 +1,63 @@
+import requests
```

Comment:
since this code mainly has data pulling from zomato you could name it as `zomato_api.py` or something similar